If TooltipInfo is created with default constructor, ContentType is not set and client side exception will occur.

If TooltipInfo is created with default constructor, ContentType is not set and client side exception will occur.

VTooltip.setTooltipText does not have null pointer check for missing content type.

And for example CalendarConnector.java has method getTooltTip which uses GWT.create to create TooltipInfo (i.e. default constructor)

```java
    @Override
    public TooltipInfo getTooltipInfo(
            com.google.gwt.dom.client.Element element) {
        TooltipInfo tooltipInfo = null;
        Widget w = WidgetUtil.findWidget(element, null);
        if (w instanceof HasTooltipKey) {
            tooltipInfo = GWT.create(TooltipInfo.class);
            String title = tooltips.get(((HasTooltipKey) w).getTooltipKey());
            tooltipInfo.setTitle(title != null ? title : "");
        }
        if (tooltipInfo == null) {
            tooltipInfo = super.getTooltipInfo(element);
        }
        return tooltipInfo;
    }
```

Thus e.g. following exception can occur.

Caused by: com.google.gwt.core.client.JavaScriptException: (TypeError) : Cannot read property 'java_lang_Enum_ordinal' of undefined
    at Unknown.com_vaadin_client_VTooltip_$setTooltipText__Lcom_vaadin_client_VTooltip_2Lcom_vaadin_client_TooltipInfo_2V(com.ideas.calendar.customcalendar.demo.DemoWidgetSet-0.js)
    at Unknown.com_vaadin_client_VTooltip$TooltipEventHandler_$handleShowHide__Lcom_vaadin_client_VTooltip$TooltipEventHandler_2Lcom_google_gwt_event_dom_client_DomEvent_2ZV(com.ideas.calendar.customcalendar.demo.DemoWidgetSet-0.js)
    at Unknown.com_vaadin_client_VTooltip$TooltipEventHandler_onMouseMove__Lcom_google_gwt_event_dom_client_MouseMoveEvent_2V(com.ideas.calendar.customcalendar.demo.DemoWidgetSet-0.js)

--- a/client/src/main/java/com/vaadin/client/TooltipInfo.java
+++ b/client/src/main/java/com/vaadin/client/TooltipInfo.java
@@ -42,6 +42,7 @@ public class TooltipInfo {
      * Constructs a new tooltip info instance.
      */
     public TooltipInfo() {
+        this("", ContentMode.PREFORMATTED);
     }
 
     /**

